config/global.ini.php

```diff
@@ -895,6 +895,9 @@
 ; NOTE: you must also set "[Tracker] debug = 1" to enable the profiler.
 enable_sql_profiler = 0
 
+; Enables using referrer spam blacklist.
+enable_spam_filter = 1
```

Comment:
@john357smith could you make that change as part of the other PR https://github.com/matomo-org/matomo/pull/15744 ?